solidity/test/unit/UpdateStorageMirrorGuard.t.sol

```diff
@@ -45,25 +50,25 @@ contract UnitUpdateStorageMirrorGuard is Base {
 
     vm.mockCall(
       address(guardCallbackModule),
-      abi.encodeCall(IGuardCallbackModule.saveUpdatedSettings, (safe, SETTINGS_HASH)),
+      abi.encodeCall(IGuardCallbackModule.saveUpdatedSettings, (safe, settingsHash)),
       abi.encode()
     );
     vm.expectCall(
-      address(guardCallbackModule), abi.encodeCall(IGuardCallbackModule.saveUpdatedSettings, (safe, SETTINGS_HASH))
+      address(guardCallbackModule), abi.encodeCall(IGuardCallbackModule.saveUpdatedSettings, (safe, settingsHash))
     );
     vm.prank(safe);
     updateStorageMirrorGuard.checkAfterExecution(_txHash, true);
 
     assertFalse(updateStorageMirrorGuard.didSettingsChange());
-    assertEq(updateStorageMirrorGuard.settingsHash(), keccak256(abi.encodePacked('')));
+    assertEq(updateStorageMirrorGuard.settingsHash(), keccak256(abi.encodePacked('')), 'Settings hash should reset');
   }
 
   function testCheckAfterExecutionNoSettingsChange(bytes32 _txHash) public {
     vm.prank(safe);
     updateStorageMirrorGuard.checkAfterExecution(_txHash, true);
 
     assertFalse(updateStorageMirrorGuard.didSettingsChange());
```

Comment:
The value to adding the reason section shines when one test has multiple asserts as its much easier to debug because it will tell you exactly which asserts failed, as this is a very simple test its ok like this imo so am approving just wanted to point this out as a note.